src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-TPCC-SYSBENCH.json

```diff
@@ -0,0 +1,165 @@
+{
+    "Description": "Sysbench TPCC MySQL Database Server Performance Workload",
+    "MinimumExecutionInterval": "00:01:00",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "04:00:00",
+        "SupportedPlatforms": "linux-x64,linux-arm64",
+        "SupportedOperatingSystems": "Debian,Ubuntu"
+    },
+    "Parameters": {
+        "Benchmark": "TPCC", 
```

Comment:
Remove "Benchmark" parameter from globals. Given the profile specifically (and rightfully) calls out "TPCC-SYSBENCH", do not let the user have an option to override the "Benchmark" on the command line. We do not intend to let the user deviate from the TPCC benchmark when using this profile.